Спринт 23

IvanPodlesnykh opened the pull request:
1. Добавлена иконка приложения
2. Реализован экран плейлиста
3. Реализованы функции в плейлисте
4. Реализован экран редактора плейлиста

IvanPodlesnykh committed:
Спринт 23
1. Добавлена иконка приложения
2. Реализован экран плейлиста
3. Реализованы функции в плейлисте
4. Реализован экран редактора плейлиста

IvanPodlesnykh committed:
Спринт 23
5. Убрал отображение нижней панели навигации на экране редактора плейлиста

kosstyango commented:
⚠️ Критические замечания:
При просмотре трека из истории поиска его кнопка "Нравится" всегда показывается в неподсвеченном состоянии (см. видео-скрин №1).
При возврате из Медиатеки на Поиск и установке курсора в поле поиска История поиска показывается "поверх" результатов поиска. (см. видео-скрин №2).

🍏 Рекомендации, не влияющие на результат проверки работы:
На экране "Поиск" заголовок "Вы искали" чуть-чуть подрезается снизу: это не сильно в глаза бросается, так что НЕкритично.


https://github.com/IvanPodlesnykh/PlaylistMaker/assets/73995013/443b29b6-86bc-4164-b539-a7db927be80b


https://github.com/IvanPodlesnykh/PlaylistMaker/assets/73995013/be18f657-fbd6-4dee-b6a6-78f50d371ca6

IvanPodlesnykh committed:
Спринт 23 правки
1. Исправил ошибку, при которой трек из истории поиска не определялся как добавленный в Понравившиеся
2. Исправил ошибку, при которой на экране поиска треков одновременно отображался список треков из поиска и история поиска
